### ISSUE TITLE
Small edge-case fix

### ISSUE THREAD

**Issue by Enemby** [JYandev/Localization-Manager#2]
Hey, so I found a small issue, and it seemed overkill to open a pull request for it, so here we are.

If a GameObject is disabled on scene load, it will not be updated to a new language, even with auto update enabled. The reason for this is the language is not set in awake, only if the language is changed.

You can fix this by adding "updateLocale();" into Awake() in LocaleText.cs . I'll attach a full script with my modification to make it easy to add.
```
using System;
using UnityEngine;
using UnityEngine.UI;

[RequireComponent(typeof(Text))]
/// <summary>
/// Provides the ability to manipulate the sibling Text component at runtime to match the current Locale.
/// </summary>
public class LocaleText : MonoBehaviour {

    [SerializeField]
    private string textID; // The ID of the string resource we want to grab.
    [SerializeField]
    private bool autoUpdate = true; // Does this UI text automatically apply changes to locale?
    private Text textComponent;
    private LocalizationManager localeManager;

    private void Awake () {
        // Cache references:
        textComponent = GetComponent<Text>();
        localeManager = GameObject.FindWithTag("Localization Manager").GetComponent<LocalizationManager>();
        // Bind event if we autoUpdate:
		updateLocale();
        if (autoUpdate == true) {
            localeManager.languageChanged += updateLocale;
        }
    }
    /// <summary>
    /// Attempts to fetch the associated string resource from the LocalizationManager.
    /// Will update the sibling Text component's text attribute if successful.
    /// </summary>
    public void updateLocale () {
        try {
            string response = localeManager.getText(textID);
            if (response != null) {
                textComponent.text = response;
            }
        }
        catch (NullReferenceException e) {
            Debug.Log(e);
        }
    }
}
```

**Comment by jpyankel** [comment]
Sorry for getting back to you so late.

I will be taking a look at this tonight. Thanks for your help!

**Comment by jpyankel** [comment]
Hello again,

I've fixed the issue by running LocalizationManager's initialization in Awake, where it should have been before. All LocaleTexts will now update themselves in their Start methods - including those which have been initially disabled.

I decided to do this instead of simply calling updateLocale in the Awake method of LocaleText because updateLocale was being called before the LocalizationManager could fully initialize; errors would be thrown by any objects that were not initially disabled (Awake does not run on initially disabled objects). While users could be instructed to set their script order to prioritize LocalizationManager's Awake, I prefer to leave script order alone for simplicity's sake. 

As a side note, I have also found that once the event is bound (such as when enabling an initially disabled object), LocaleTexts will receive updates even when disabled afterwards. I have added an "Initially Disabled Text" object to the ExampleScene to provide a convenient way to test this new functionality.

Commit de4056495bfee99bbaee35867b062ef2f37a4d1d closes #2
